ground-server/src/components/actuation-box.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import React, { useState } from 'react';
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "@/components/ui/alert-dialog"
+
+import { Button } from "@/components/ui/button";
+import { useTheme } from "next-themes";
+
+interface ActuationBoxProps {
+  title: string;
+}
+
+export function ActuationBox({ title }: ActuationBoxProps) {
+  const { theme } = useTheme(); 
+  const [isOn, setIsOn] = useState(false); //default state is off
+
+  const handleTurnOn = () => {
+    setIsOn(true);
+  };
+
+  const handleTurnOff = () => {
+    setIsOn(false);
+  };
+
+  return (
+    <div className="border border-gray-300 p-6 rounded-lg bg-white dark:bg-black text-center">
+      <h2 className="text-lg font-bold mb-2">{title}</h2>
+      <div>
+        <h4 className="text-sm font-medium mb-2">Current Telemetry</h4>
+        <div 
+          className={`mb-4 h-[100px] ${theme === "dark" ? "bg-gray-800" : "bg-gray-200"}`}>
```

Comment:
Try not to do this, instead replace `${theme === "dark" ? "bg-gray-800" : "bg-gray-200"}` with `bg-gray-200 dark:bg-gray-800`. I think this was the cause of the flickering and wrong color during code review